src/main/resources/db/changelog/changes/v1.0.3/sql/create-list-versions-table.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS list_versions
+(
+    id                      UUID            NOT NULL PRIMARY KEY,
+    name                    VARCHAR(255)    NOT NULL,
+    description             VARCHAR(1024),
+    fql_query               text,
+    user_friendly_query     text,
+    fields                  VARCHAR[],
+    updated_by              UUID            NOT NULL,
+    updated_by_username     VARCHAR(1024)     NOT NULL,
```

Comment:
should this be varchar or text?

Reply:
this should be varchar as every where we do 'varchar' for names